src/app/shared/helpers/duration.ts

```diff
@@ -4,6 +4,7 @@ export const MINUTES = 60 * SECONDS;
 export const HOURS = 60 * MINUTES;
 export const DAYS = 24 * HOURS;
 export const MONTHS = 30 * DAYS;
+export const YEARS = 12 * MONTHS;
```

Comment:
probably that's more correct to say a year is 365 days than saying that it is 12 months of 30 days.

Reply:
fixed